frontend/src/pages/Projects/Project/ProjectDetails/BudgetSection/FormConfig.ts

```diff
@@ -105,6 +110,30 @@ const getRecoveredTotalsByQuarter = async (
   setFieldValue("total", toCurrency);
 };
 
+const getResponsabilityServiceLine = async ({
+  newValue,
+  setFieldValue,
+}: {
+  newValue: { value: string | number };
+  setFieldValue: Function;
+}) => {
+  const getCall = async () => {
+    const results = await apiAxios()
+      .get(`/project/budget/responsibilityservice/${newValue?.value}`)
```

Comment:
If newValue can be empty here what URL would this request be sent to? Just wondering if this would cause another `/undefined` thing. The function could be changed to check if newValue.value exists first and just return 0 or whatever a reasonable return would be so it doesn't even try to make the request.

Reply:
This function only gets called when there is a value for newValue... It's a custom handler.  If you follow the chain you'll see it.  But I can add a check one second.